gramps/gen/const.py

```diff
@@ -99,10 +99,12 @@
     HOME_DIR = os.path.join(USER_HOME, "gramps")
 elif "USERPROFILE" in os.environ:
     USER_HOME = get_env_var("USERPROFILE")
-    if "APPDATA" in os.environ:
-        HOME_DIR = os.path.join(get_env_var("APPDATA"), "gramps")
-    else:
-        HOME_DIR = os.path.join(USER_HOME, "gramps")
+    HOME_DIR = os.path.join(GLib.get_user_config_dir(), "gramps")
+    if not os.path.exists(HOME_DIR):
+        if "APPDATA" in os.environ:
```

Comment:
@Nick-Hall To use the existing configuration only if there are any databases present, I would add a check for contents under grampsdb in both cases of this `if`, and modify `HOME_DIR` only if there are any contents present (minimal check for any entries in that directory, no validation that they are actual, valid databases). If there are no databases, `HOME_DIR` would be unchanged from the setting in L102 above.

This should achieve the intent you mentioned.

Reply:
Implemented in the commit 8648ecf279d0e21f750d0ceeea46abb47a21788e